buildpacks/java/tests/detect_test.go

```diff
@@ -62,7 +62,7 @@ func testDetect(t *testing.T, when spec.G, it spec.S) {
 
 	when("BP_FUNCTION is configured without func.yaml", func() {
 		it.Before(func() {
-			Expect(os.Setenv("BP_FUNCTION", "function.Handler")).To(Succeed())
+			t.Setenv("BP_FUNCTION", "function.Handler")
```

Comment:
Not part of python, but came in as part of a global replace across all buildpacks to ensure tests that don't contaminate each other. It's small so I figured I'd leave it, but I can separate if requested.